bower.json

```diff
@@ -9,6 +9,8 @@
     "purescript-nonempty": "^5.0.0",
     "purescript-aff": "^5.0.2",
     "purescript-halogen": "^4.0.0",
+    "purescript-halogen-bootstrap4": "^0.1.2",
```

Comment:
Glad to see `RemoteData` in here! But I'd rather not use the Halogen Bootstrap dependency.